tests/integration/relations/pgbouncer_provider/test_pgbouncer_provider.py

```diff
@@ -532,6 +532,9 @@ async def test_indico_datatabase(ops_test: OpsTest) -> None:
             timeout=2000,
         )
 
+        # FIXME: Pebble in Indico doesn't survive the subsequent restart. Investigate further
+        await ops_test.model.remove_application("indico", block_until_done=True)
+
```

Comment:
Created [DPE-6331](https://warthogs.atlassian.net/browse/DPE-6331) to investigate further.